src/main.cpp

```diff
@@ -200,14 +200,26 @@ void publishMsgs(um6::Registers& r, ros::NodeHandle* imu_nh, sensor_msgs::Imu& i
     imu_msg.orientation_covariance[8] = r.covariance.get_scaled(15);
```

Comment:
Does the flag to use NED instead of converted ENU not apply to the orientation portion of the message?